etl/src/main/java/org/pih/hivmigration/etl/sql/ContactsMigrator.groovy

```diff
@@ -0,0 +1,127 @@
+package org.pih.hivmigration.etl.sql
+
+/**
+ * Migrates Intake form section 11, "Contacts" to the HIV Intake form.
+ *
+ * Data comes from HIV_CONTACTS / hivmigration_contacts, which is imported by StagingTablesMigrator.
+ */
+class ContactsMigrator extends ObsMigrator {
+    @Override
+    def void migrate() {
+        executeMysql("Create temporary table for contact person groups", '''
+            CREATE TABLE hivmigration_tmp_contacts (
+                obs_group_id INT PRIMARY KEY AUTO_INCREMENT,
+                source_encounter_id INT,
+                first_name VARCHAR(64),
+                last_name VARCHAR(64),
+                old_relationship VARCHAR(76),
+                relationship_uuid VARCHAR(38),
+                age INT,
+                death_uuid VARCHAR(38),
+                hiv_status_uuid VARCHAR(38)
+            );
+        ''')
+
+        setAutoIncrement("hivmigration_tmp_contacts", "(select max(obs_id)+1 from obs)")
+
+        executeMysql("Populate temp table of patient contacts", '''
+            INSERT INTO hivmigration_tmp_contacts
+                (source_encounter_id, first_name, last_name, old_relationship, relationship_uuid, age, death_uuid, hiv_status_uuid)
+            SELECT he.source_encounter_id,
```

Comment:
See my comment on the ticket @brandones .  We should do a distinct query of contacts here, grouping on patient_id, in order to eliminate duplicate rows of contacts, and to see what the true maximum we need to support on the form is.

Reply:
Seems to make sense... @brandones is there thoughts on why this might not work?

Reply:
Oh, it will be fine. We won't worry about the cases where there are different spellings (they will appear as separate contacts). We can do `MAX` on all the other fields, and if there's a discrepancy we can issue a data warning about it.